GSoC-2025/movement_support for Kalman filters_(Shraddha Bharadwaj).md

```diff
@@ -0,0 +1,125 @@
+
+
+## Project title
+
+movement: support for Kalman filters (Shraddha Bharadwaj)
+
+## Personal details
+Please include the following information:
```

Comment:
Note that we also require a link to a code contribution (a pull request to one of our projects, on another open-source project).

Reply:
Copied from our [guidelines](https://neuroinformatics.dev/get-involved/gsoc/guidelines.html):

> You will need to submit a pull request (ideally to one of the NIU projects) as part of your application. It does not have to be accepted - the goal is to show that you know how git, GitHub, pull requests and code reviews work. This also allows mentors to evaluate your application based on a real code contribution. If you have previously contributed to an NIU or other open source project, you can point to those pull requests in your application too.